tests/functional/tests/base.py

```diff
@@ -15,6 +15,8 @@
 class BaseTestCase(unittest.TestCase):
     """Base class for functional testing of verifiers."""
 
+    loop = asyncio.get_event_loop()
```

Comment:
Is this 100% safe approach? the `asyncio.get_event_loo()` gets executed at the moment when this module is first imported (and never again). Will this class attribute have correct reference to the current event loop?
Maybe it'd be better to implement it as a caching property?